tensorboard/webapp/feature_flag/views/BUILD

```diff
@@ -0,0 +1,33 @@
+load("//tensorboard/defs:defs.bzl", "tf_ng_module", "tf_sass_binary")
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])
+
+tf_sass_binary(
+    name = "style",
+    src = "feature_flag_page_style.scss",
+    deps = ["//tensorboard/webapp:angular_material_sass_deps"],
+)
+
+tf_ng_module(
+    name = "feature_flag_page",
+    srcs = [
+        "feature_flag_page_component.ts",
+        "feature_flag_page_container.ts",
+        "feature_flag_page_module.ts",
```

Comment:
I think it would be ok to just name this feature_flag_module, with the assumption that if other files are added to the view they are likely to be included in the same module.

Reply:
Done.